products/stratos/modules/distribution/src/main/conf/drools/mincheck.drl

```diff
@@ -83,9 +81,8 @@ dialect "mvel"
                                  0, 0, 0, 0, 0, 0,0, 0, 0,
                                  clusterInstanceContext.getMinInstanceCount(), 0,
                                  additionalInstances, scalingReason);
-        } else {
-            log.warn("Scaling decision publisher is not enabled");
         }
+
```

Comment:
That's OK. We will have to display the warning message in the start up when we are checking whether that DAS monitoring and metering enable.